libs/sdk-ui-kit/src/Dialog/ConfirmDialogBase.tsx

```diff
@@ -67,31 +68,34 @@ export class ConfirmDialogBase extends DialogBase<IConfirmDialogBaseProps> {
                     <div className="gd-dialog-content">{children}</div>
 
                     <div className="gd-dialog-footer">
-                        {showProgressIndicator && <LoadingSpinner className="gd-dialog-spinner small" />}
+                        <div className="gd-dialog-footer-left-content">{footerLeftRenderer?.()}</div>
```

Comment:
Should this div be rendered if there is no `footerLeftRenderer`?

Reply:
Yes this was my prev impl but problem is than I needed different flex prop (space-between vs flex-end) in storybook looks everything well but backstop did not (was still flex-end).